.github/actions/check-package-version/index.js

```diff
@@ -0,0 +1,27 @@
+const core = require('@actions/core');
+const fs = require('fs');
+
+const encoding = 'utf8';
+
+try {
+    const workspace = core.getInput('workspace');
+    const releaseVer = getReleaseVersion();
+    
+    fs.readFile(`${workspace}/package.json`, {encoding: encoding}, (err, data) => {
+        if (err) throw err;
```

Comment:
Eslint will fail on single-statement blocks, expand using braces.

Reply:
Should I be using `eslint:all` instead of `eslint:recommended`?

Reply:
I thought that was the default?

We use recommended on cylc-ui: https://github.com/cylc/cylc-ui/blob/master/.eslintrc.js